docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: "2.4"
+services:
+ lform-app:
+    build: .
+    container_name: lform
+    restart: on-failure
+    ports:
+      - "${HTTP:-8080}:4200"
+    healthcheck:
+      test: ["CMD", "cd /app && yarn healthcheck"]
```

Comment:
What does yarn healthcheck do?

Reply:
After you start the container with `docker-compose up -d ` when you do `docker-compose ps` it says if the container is starting or up, for some reason it in that case it says unhealthy but without concequences. 

Of couse you can use docker iso docker-compose but the cli is less user friendly that why i always add the compose file to the Dockerfile.